loopd/swapclient_server.go

```diff
@@ -646,10 +646,8 @@ func (s *swapClientServer) SuggestSwaps(ctx context.Context,
 
 	for _, swap := range swaps {
 		loopOut = append(loopOut, &looprpc.LoopOutRequest{
-			Amt: int64(swap.Amount),
-			OutgoingChanSet: []uint64{
-				swap.Channel.ToUint64(),
-			},
+			Amt:             int64(swap.Amount),
+			OutgoingChanSet: swap.OutgoingChanSet,
```

Comment:
I now realize that the returned `LoopOutRequest` messages in master aren't directly usable because there are no fees set.

Maybe nice to do a quick test modification locally where you pass in the response of `SuggestSwaps` to `LoopOut` to see that it actually works.

In terms of commit structure, you could also choose to introduce the default values and just assign them here. Then the 'bug' would be fixed first. Further commits add new functionality.

Reply:
Wasn't really thinking of swap suggestions as something that would be passed straight to the rpc tbh, was more thinking that people would use them for input into the cli. But nice idea, moved the defaults forward and added them to the output in the first commit.